test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -23,8 +28,14 @@
     countParam: $measurePods
     collectMetrics: true
   workloads:
+  - name: 5Nodes
```

Comment:
did you consider 50 instead? How much more time does it need?

Reply:
I would take slightly longer (~10% running locally), but would it be beneficial?

Reply:
I can't think of a problem that would show at 50 but not at the 5 scale, so let's keep it like this.